### ISSUE TITLE
[gui error report] SyntaxError: C:\Users\Administrator\.ipfs\config: Unexpected token T in 

### ISSUE THREAD

**Issue by Brent-Sanchez** [ipfs/ipfs-desktop#2773]
👉️ Please describe what you were doing when this error happened.

**Specifications**

- **OS**: win32 10.0.19045
- **IPFS Desktop Version**: 0.33.0
- **Electron Version**: 19.1.9
- **Chrome Version**: 102.0.5005.167

**Error**

```
SyntaxError: C:\Users\Administrator\.ipfs\config: Unexpected token T in JSON at position 2088
    at JSON.parse (<anonymous>)
    at Object.readFileSync (C:\Program Files\IPFS Desktop\resources\app.asar\node_modules\jsonfile\index.js:52:17)
    at readConfigFile (C:\Program Files\IPFS Desktop\resources\app.asar\src\daemon\config.js:68:13)
    at checkPorts (C:\Program Files\IPFS Desktop\resources\app.asar\src\daemon\config.js:332:18)
    at getIpfsd (C:\Program Files\IPFS Desktop\resources\app.asar\src\daemon\daemon.js:71:16)
    at async startDaemon (C:\Program Files\IPFS Desktop\resources\app.asar\src\daemon\daemon.js:210:17)
    at async startIpfs (C:\Program Files\IPFS Desktop\resources\app.asar\src\daemon\index.js:49:17)
    at async setupDaemon (C:\Program Files\IPFS Desktop\resources\app.asar\src\daemon\index.js:116:3)
    at async Promise.all (index 1)
    at async run (C:\Program Files\IPFS Desktop\resources\app.asar\src\index.js:68:5)
```

**Comment by SgtPooki** [comment]
I believe there is invalid json in your config file. Can you share your config file contents from `C:\Users\Administrator\.ipfs\config`?

**Comment by Brent-Sanchez** [comment]
> I believe there is invalid json in your config file. Can you share your config file contents from `C:\Users\Administrator\.ipfs\config`?

I'm sorry for the no-reply man, this is from a work around account profile.  I had some roadblocks in my normal windows account that were blocking code execution and not notifying me.  I've since got it together, deleted the kubo/webview stuff out of that built in admin account and moved on.  I've got everything running smoothly now.  Sorry again.  :-D
